docs/sources/administration/configuration.md

```diff
@@ -772,6 +772,22 @@ Refer to [Auth proxy authentication]({{< relref "../auth/auth-proxy.md" >}}) for
 
 Refer to [LDAP authentication]({{< relref "../auth/ldap.md" >}}) for detailed instructions.
 
+## [aws]
+
+In this section, core and external AWS plugins can be configured.
+
+### allowed_auth_providers
+
+Specify what authentication providers should be allowed for AWS plugins. The providers will be listed in the data source configuration page for the plugin. If the plugin is configured using provisioning, only providers that are specified in `allowed_auth_providers` will be accepted.
```

Comment:
```suggestion
Specify what authentication providers that the AWS plugins allow. For a list of allowed providers, refer to the data-source configuration page for a given plugin. If you configure a plugin by provisioning, only providers that are specified in `allowed_auth_providers` are allowed.
```
* Remove _should_ to reduce ambiguity. Alternatively, use SHOULD in all caps to make it clear that you are referring to reserved words SHOULD, MUST, etc. per https://tools.ietf.org/html/rfc2119. This can, however, can come across as yelling.
* Remove future tense unless there is a (rare) specific reason to use it: _will be_ --> _is_.
* Notice that the passive at end of this suggestion works well because the object is the center of attention in this phrase, "only providers that are specified in `allowed_auth_providers` are allowed."
* Change "accepted" to "allowed" because it seems like those are simply two verbs to refer to the same thing.